tests/lib/nm/ipv4_test.py

```diff
@@ -24,7 +25,7 @@
 
 @pytest.fixture
 def NM_mock():
-    with mock.patch.object(nm.ipv4.nmclient, 'NM') as m:
+    with mock.patch.object(nm.nmclient, 'NM') as m:
```

Comment:
We need to mock at the usage not at the original definition.
These change also also not related to this PR/commit, you can propose it in another context if you think it is needed.

Reply:
Sure. Just old left up. Will clean them up.